packages/muelu/src/CMakeLists.txt

```diff
@@ -356,6 +356,9 @@ APPEND_GLOB(SOURCES */*.cpp)
 LIST(REMOVE_ITEM SOURCES ${DIR}/Graph/MueLu_LinkedList.cpp)
 SET(SOURCES_LGN ${MUELU_ETI_CPP_SOURCES_LGN})
 APPEND_GLOB(SOURCES_LGN ${DIR}/Graph/MueLu_LinkedList.cpp)
+APPEND_GLOB(SOURCES_LGN "${DIR}/MueCentral/*.cpp")
+APPEND_GLOB(SOURCES_LGN "${DIR}/Utils/*.cpp")
```

Comment:
This is adding `*.cpp` files to SOURCES_LGN that have already been added to SOURCES.  The result is that the same object files will appear in both `libmuelu.a` and `libmuelu_lgn.a`.